Update `README.md`. (#180)
yiisoft/db-sqlite: README.md

    <br>
</p>

> **Yii DataBase SQLite Extension** is a package for working with [SQLite] databases in PHP. It is a part of the [YiiFramework], which is a high-performance, component-based framework for developing modern web applications.

Reviewer: ```suggestion
**Yii Database SQLite Extension** is a package for working with [SQLite] databases in PHP. It is a part of the [YiiFramework], which is a high-performance, component-based framework for developing modern web applications.
```

> **Yii DataBase SQLite Extension** package provides a set of classes for interacting with [SQLite] databases in PHP. It includes a database connection class, a command builder class, and a set of classes for representing database tables and rows as PHP objects.

Reviewer: ```suggestion
The package provides a set of classes for interacting with [SQLite] databases in PHP. It includes a database connection class, a command builder class, and a set of classes for representing database tables and rows as PHP objects.
```

> Using the **Yii DataBase SQLite Extension** package, you can perform a variety of tasks with [SQLite] databases in PHP, such as connecting to a database, executing SQL queries, and working with database transactions. You can also use it to create and manipulate database tables and rows, and to perform advanced database operations such as joins and aggregates.

Reviewer: ```suggestion
You can perform a variety of tasks with [SQLite] databases in PHP, such as connecting to a database, executing SQL queries, and working with database transactions. You can also use it to create and manipulate database tables and rows, and to perform advanced database operations such as joins and aggregates.
```

Overall, **Yii DataBase SQLite Extension** is a powerful tool for working with [SQLite] databases in PHP, and is well-suited for use in web applications built with the [YiiFramework].

It is used in [YiiFramework] but can be used separately.

[SQLite]: https://www.sqlite.org/

> [YiiFramework]: https://github.com/yiisoft/core

Reviewer: ```suggestion
[Yii Framework]: https://www.yiiframework.com
```
